dom.bs

```diff
@@ -5875,19 +5875,19 @@ dictionary ShadowRootInit {
 initialized.
 
 <p>An <a for=/>element</a>'s <a for=Element>custom element state</a> is one of
-"<code>undefined</code>", "<code>failed</code>", "<code>uncustomized</code>", or
-"<code>custom</code>". An <a for=/>element</a> whose <a for=Element>custom element state</a> is
-"<code>uncustomized</code>" or "<code>custom</code>" is said to be
-<dfn export id=concept-element-defined for=Element>defined</dfn>. An <a for=/>element</a> whose
-<a for=Element>custom element state</a> is "<code>custom</code>" is said to be
-<dfn export id=concept-element-custom for=Element>custom</dfn>.
+"<code>undefined</code>", "<code>failed</code>", "<code>uncustomized</code>",
+"<code>precustomized</code>", or "<code>custom</code>". An <a for=/>element</a> whose <a
+for=Element>custom element state</a> is "<code>uncustomized</code>", "<code>precustomized</code>",
+or "<code>custom</code>" is said to be <dfn export id=concept-element-defined
+for=Element>defined</dfn>. An <a for=/>element</a> whose <a for=Element>custom element state</a> is
+"<code>custom</code>" is said to be <dfn export id=concept-element-custom for=Element>custom</dfn>.
 
 <p class=note>Whether or not an element is <a for=Element>defined</a> is used to determine the
 behavior of the '':defined'' pseudo-class. Whether or not an element is <a for=Element>custom</a> is
 used to determine the behavior of the <a href=#mutation-algorithms>mutation algorithms</a>. The
-"<code>failed</code>" state is used to ensure that if a <a>custom element constructor</a> fails to
-execute correctly the first time, it is not executed again by an
-<a lt="upgrade an element">upgrade</a>.</p>
+"<code>failed</code>" and "<code>precustomized</code>" states are used to ensure that if a <a>custom
```

Comment:
No newlines in inline elements. Sorry that this is different from HTML.

Reply:
Ahh ok. Fixed.